src/components/Avatar/Example.md

```diff
@@ -72,7 +72,7 @@ const image = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJAAAACQCAMAAADQmBK
 class Badge extends React.Component {
   constructor() {
     this.state = {
-      mails: ~~(Math.random() * 10)
+      mails: 10
```

Comment:
Its okay but does not cases with 1 digit. The badge should look fine for at least 1 and 2 digits.